lib/routes/bitget/announcement.ts

```diff
@@ -0,0 +1,187 @@
+
+        case 'new-announcement':
+            reqBody.stationLetterType = '06';
+            break;
+
+        case 'all':
+            reqBody.stationLetterType = '0';
+            reqBody.excludeStationLetterType = '00';
+            break;
+
+        default:
+            throw new Error('Invalid type');
+    }
+
+    const response = (await cache.tryGet(`bitget:announcement:${type}:${pageSize}:${lang}`, async () => {
+        const result = await ofetch<BitgetResponse>(announcementApiUrl, {
+            method: 'POST',
+            body: reqBody,
+            headers,
+        });
+        if (result?.code !== '200') {
+            throw new Error('Failed to fetch announcements, error code: ' + result?.code);
+        }
+        return result;
+    })) as BitgetResponse;
+
+    if (!response) {
+        throw new Error('Failed to fetch announcements');
+    }
+    const items = response.data.items;
+    const data = await Promise.all(
+        items.map(
+            (item) =>
+                cache.tryGet(`bitget:announcement:${item.id}:${pageSize}:${lang}`, async () => {
+                    // 从 unix 时间戳转换为日期
+                    const date = dayjs.unix(Number(item.sendTime?.slice(0, 10))).toDate();
+                    const reqItem: DataItem = {
+                        title: item.title ?? '',
+                        link: item.openUrl ?? '',
+                        pubDate: item.sendTime ? date : undefined,
+                        description: item.content ?? '',
+                    };
+
+                    if (item.imgUrl) {
+                        reqItem.image = item.imgUrl;
+                    }
+
+                    if (item.stationLetterType === '01' || item.stationLetterType === '06') {
+                        try {
+                            const itemResponse = await ofetch<string>(item.openUrl ?? '', {
+                                headers,
+                            });
+                            const $ = load(itemResponse);
+                            const content = $('div > div.ArticleDetails_actice_details_main__oIjfu');
+                            reqItem.description = content.html() || item.content || '';
+                        } catch {
+                            throw new Error('Failed to fetch announcement details');
+                        }
+                    }
+                    return reqItem;
+                }) as Promise<DataItem>
+        )
+    );
+
+    return {
+        title: `Bitget | ${find_type_label(type)}`,
+        link: `https://www.bitget.com/${languageCode.replace('_', '-')}/inmail`,
+        item: data,
+    };
+};
+
+const find_type_label = (type: string) => {
+    const typeMap = {
+        all: 'All',
+        'new-listing': 'New Listing',
+        'latest-activities': 'Latest Activities',
+        'new-announcement': 'New Announcement',
+    };
+    return typeMap[type];
+};
+
+export const route: Route = {
+    path: '/announcement/:type/:pageSize?/:lang?',
```

Comment:
Don't place page size as a path parameter. This should use the global parameter [`limit`](https://docs.rsshub.app/guide/parameters#limit-entries) instead.
https://github.com/DIYgod/RSSHub/blob/28f2d3e19ebeb67ebe0260713eba1dac6691e58c/lib/middleware/parameter.ts#L283